cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -88,6 +88,41 @@ users:
   - system:serviceaccount:rook-ceph:rook-csi-cephfs-plugin-sa
   - system:serviceaccount:rook-ceph:rook-csi-cephfs-provisioner-sa
 ---
+# Rook Ceph Operator Config
+# Use this ConfigMap to override operator configurations
+# Precedence will be given to this config in case
+# Env Var also exists for the same
+#
+# kind: ConfigMap
+# apiVersion: v1
+# metadata:
+#   name: rook-ceph-operator-config
+#   # should be in the namespace of the operator
+#   namespace: rook-ceph
+# data:
```

Comment:
How about creating the configmap by default, but just leave all the settings under `data` commented out?

Reply:
Done.